dokumentlager/domene/src/main/java/no/nav/foreldrepenger/melding/beregning/BeregningsresultatFP.java

```diff
@@ -20,13 +20,13 @@ public static Builder ny() {
 
 
     public static final class Builder {
-        private List<BeregningsresultatPeriode> beregningsresultatPerioder;
+        private List<BeregningsresultatPeriode> beregningsresultatPerioder = new ArrayList<>();
 
         private Builder() {
         }
 
         public Builder medBeregningsresultatPerioder(List<BeregningsresultatPeriode> beregningsresultatPerioder) {
-            this.beregningsresultatPerioder = beregningsresultatPerioder;
```

Comment:
liker denne måten å gjøre det på.. Kunne endra navnet til leggTil.. men whatever

Reply:
Enig i det. Har bare brukt builder-generator kjørt 'replace all with -> med'.
Sikkert masse rom for utbedringer generelt :)